### ISSUE TITLE
Reformat en-US style guide to incorporate style guides from SUMO and other areas of Mozilla

### ISSUE THREAD

**Issue by gueroJeff** [mozilla-l10n/styleguides#70]
The en-US style guide needs to be restructured to ensure that we can add content from various other areas of Mozilla. Off the top of my head, that would mean establishing everything we have currently under a h2 "Products and web pages" and allow other areas of Mozilla to follow suit (e.g., h2 "Translating articles for SUMO knowledge base" with content following it).

**Comment by flodolo** [comment]
#53 is still waiting for another pair of eyes, and IMO it's a blocker for any further changes

**Comment by gueroJeff** [comment]
Good point. Let's add it to the PM Group weekly call and make it clear to
everyone to review it before then to discuss.

On Thu, Jan 25, 2018 at 11:58 AM, Francesco Lodolo <notifications@github.com

**Comment by MikkCZ** [comment]
#53 is merged for some time. What is the next move for SUMO style guides?
